Tests/AsyncQueueTests/FIFOQueueTests.swift

```diff
@@ -36,46 +36,46 @@ final class FIFOQueueTests: XCTestCase {
 
     // MARK: Behavior Tests
```

Comment:
There are at least a couple places in the file like this that we should clean up.
https://github.com/dfed/swift-async-queue/blob/3454ea6497aeb917b6fe084efd69d95d9c7cff35/Tests/AsyncQueueTests/FIFOQueueTests.swift#L81